Loop/Views/SettingsView.swift

```diff
@@ -167,9 +168,9 @@ fileprivate struct LargeButton: View {
 
     // TODO: The design doesn't show this, but do we need to consider different values here for different size classes?
     static let spacing: CGFloat = 15
-    static let imageWidth: CGFloat = 48
-    static let imageHeight: CGFloat = 48
-    static let topBottomPadding: CGFloat = 20
+    static let imageWidth: CGFloat = 60
+    static let imageHeight: CGFloat = 60
```

Comment:
[nit] should accessibility be considered for this image? It is already pretty large, but thought I should ask.

Reply:
I'm going to assume it is large enough... and count on design review to catch that if need be.